[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\victo\\AppData\\Local\\Temp\\quarto-session5094d0c45045cff8'\n"]
